test/unit/http/DataQueryEndpoints.test.js

```diff
@@ -409,14 +396,14 @@ describe('DataQueryEndpoints', () => {
                 )
             })
 
-            it('responds 500 and error message if networkNode signals error', (done) => {
-                networkNode.requestResendRange = () => intoStream.object(Promise.reject(new Error('error')))
+            it('responds 500 and error message if networkNode signals error', async () => {
+                networkNode.requestResendRange = jest.fn(() => intoStream.object(Promise.reject(new Error('expected error'))))
```

Comment:
What's the reasoning behind wrapping the function with `jest.fn` if it not being asserted (e.g. num of calls)?

Reply:
just consistency really, `jest.fn` also signals (to me at least) that this is a mock function.